fendermint/app/options/src/lib.rs

```diff
@@ -87,15 +80,30 @@ pub struct Options {
     #[arg(short, long, default_value = "dev")]
     pub mode: String,
 
-    /// Set the logging level.
+    /// Set the logging level of the console.
     #[arg(
         short = 'l',
         long,
         default_value = "info",
         value_enum,
-        env = "LOG_LEVEL"
+        env = "FM_LOG_LEVEL",
+        help = "Standard log levels, or a comma separated list of filters, e.g. 'debug,tower_abci=warn,libp2p::gossipsub=info'",
+        value_parser = parse_log_level,
+    )]
+    log_level: LogLevel,
+
+    /// Fallback for the `log_level` with a legacy env var name.
+    #[arg(hide = true, value_enum, env = "LOG_LEVEL", value_parser = parse_log_level,)]
+    _log_level: Option<LogLevel>,
```

Comment:
Do we need to keep legacy env around? Maybe we can just remove them now, otherwise they become much difficult to get rid off

Reply:
I don't know, I didn't want to throw a monkey wrench in Fluence's deployment scripts without giving some heads up. I figured it can't get more difficult, we just remove this line, it's not like any code depends on the field itself. It would be nice if there were env aliases supported by `clap`, but as far as I saw there can be only one name.